packages/exemption-crd/zarf.yaml

```diff
@@ -0,0 +1,16 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-core-exemption-crd
```

Comment:
I'm tempted to just call this "uds-crds". I don't know if we want to add the `Package` CRD here right now, but in theory this is more flexibility to provide other pre-core custom resource down the line without need to rename. Maybe that's more confusing if we aren't including the `Package` CRD right now though....thoughts?

Reply:
i like that idea. would actually have helped with an issue i ran into yesterday testing something. we could go ahead and include the Package CRD (no reason not too that i can think of).